content/en/basics/interplanetary-consensus/overview/index.md

```diff
@@ -13,45 +13,38 @@ weight: 100
 toc: true
 ---
 
-## The problem with consensus
-
 Many blockchain networks, including Filecoin, require all validators to process _all_ transactions. This causes a major bottleneck for the network and makes it difficult to increase network performance by scaling out. On top of that, different applications have different performance and security requirements, making it difficult for a single consensus layer to accommodate any type of web3 application.
```

Comment:
```suggestion
Many blockchain networks, including Filecoin, require that validators process all transactions. This causes a major bottleneck for the network, and makes it difficult to increase network performance by scaling out. On top of that, different applications have different performance and security requirements, making it difficult for a single consensus layer to accommodate the needs of all Web3 applications.
```

Reply:
```suggestion
Many blockchain networks, including Filecoin, require that all validators process all transactions. This causes a major bottleneck for the network, and makes it difficult to increase network performance by scaling out. On top of that, different applications have different performance and security requirements, making it difficult for a single consensus layer to accommodate the needs of all Web3 applications.
```

Your suggestion with a single tweak: added back the `all` validators is an essential part. The issue is not that all transactions need to be processed by someone, but that all transactions need to be processed by everyone. Even if you could interpret your simplified phrasing that way, I don't think it would be clear to all readers.